serving/samples/jib-app-java/sample.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: serving.knative.dev/v1alpha1
+kind: Service
+metadata:
+  name: jib-sample-app
+  namespace: default
+spec:
+  runLatest:
+    configuration:
+      build:
+        serviceAccountName: riff-build
+        source:
+          git:
+            url: https://github.com/knative/docs.git
+            revision: master
+        template:
+          name: jib-maven
+          arguments:
+          - name: IMAGE
+            value: &image DOCKER_REPO_OVERRIDE/helloworld-java
+          - name: DIRECTORY
+            value: serving/samples/helloworld-java
+      revisionTemplate:
+        spec:
+          container:
+            image: *image
```

Comment:
I don't recall whether we're still preferring yaml object references, or we're preferring to duplicate the string twice. YAML object references are somewhat magical, so it's possible that we're not using them in our examples. Check some other build samples or ask @samodell.

Reply:
same comment. everything here is similar to the dotnet example